### PR TITLE
Email dev after their first usage of the API

### PR DESCRIPTION
**Pull request by whytheplatypus**

### CODE REVIEW

File: apps/dot_ext/signals.py [hunk 1]
```diff
@@ -29,4 +32,35 @@ def check_first_application(sender, instance=None, created=False, **kwargs):
...
+    if instance.application.first_active is not None:
+        return
+
+    if Token.objects.filter(application=instance.application).exists():
+        return
+
+    if ArchivedToken.objects.filter(application=instance.application).exists():
+        return
+
+    if Application.objects.filter(user=instance.application.user).count() != 1:
```
Comment (dtisza1) [inline apps/dot_ext/signals.py]: What if the user has created several applications, but none have been used yet?

**Review by dtisza1 — CHANGES_REQUESTED** [review]
Looks good to me except for the one question in comments.

I'm wanting to change a few things based on your work:
* Change the check_first_application naming to outreach_first_application.
* Change the sender='dot_ext.Application' to sender='Application'.

File: apps/dot_ext/signals.py (hunk 1 above)
Comment (whytheplatypus) [reply apps/dot_ext/signals.py]: If I remember correctly that was an edge case we decided to leave out for now.

**Review by dtisza1 — APPROVED** [review]

**Review by dtisza1 — APPROVED** [review]
This looks good to me.

**Pull request merged**